src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -275,7 +275,7 @@ public function nodeCreate($node) {
 
   /**
    * Parse multi-value fields. Possible formats:
-   *    A, B, C
+   *    A, B, "C, D"
```

Comment:
This is very cryptic as documentation. Let's instead add a few lines of written text that clearly explain what is possible here.